### DISCUSSION TITLE
支持同时访问两个hadoop/hive 集群？

### DISCUSSION THREAD

**Discussion by melin** [apache/kyuubi#5545] [Q&A]
数据平台升级，为了验证升级后，加工数据是否一致，需要并行验证，在两个hadoop（华为mrs）集群运行，例如同时产出数据表：A，通过spark sql 程序同时访问两个集群 表A join操作，验证数据内容是否完全一致。借助 kyuubi 运行spark sql。
这种方式能够配置两个hive catalog？每个集群开启了kerberos 认证。在一个spark session 中同时访问这两个catalog 的表吗？
https://kyuubi.readthedocs.io/en/v1.7.1-rc0/connector/spark/hive.html

**Comment by pan3793** [comment]
It depends on lots of details of your setup, such broad questions usually won't get answered since it may require writing a book to thoroughly explain all the details. Just try, and report the exact issues you encountered.
